source/develop/dev-process/working-with-gerrit.html.md

```diff
@@ -101,80 +101,82 @@ Or alternatively without need of authentication:
       git commit --amend -s
        
 
-### Configure git personal settings
+### Configuring Git Personal Settings
 
       git config --global user.name "John Doe"
       git config --global user.email johndoe@example.com
        
 
-### Configure the commit template
+### Configuring the Commit Template
 
       cd ovirt-engine
       git config commit.template config/engine-commit-template.txt
        
 
-### Configure git commit spell check, syntax highlighting and maxwidth
+### Configuring the Git Commit Spell Check, Syntax Highlighting and Maxwidth
 
       # edit ~/.vimrc
       syntax on
       filetype plugin indent on
       autocmd Filetype gitcommit spell textwidth=72
        
 
-### Rebase
+### Performing Rebase
 
       git fetch -v
       git rebase origin/master
        
 
-### Push your patch for review
+## Pushing a Patch for Review
 
-### push/update the patch as a DRAFT
+### Pushing or Updating a Patch as a Draft
```

Comment:
@mykaul @sandrobonazzola The Gerrit user guide says that this feature will ve removed soon.
https://gerrit-review.googlesource.com/Documentation/intro-user.html > Working with Drafts.
Shall we remove this info from the wiki?